resources/assets/sass/thema/Koevoet.scss

```diff
@@ -0,0 +1,136 @@
+@font-face {
+  font-family: "Algol";
+  src: url("../../fonts/Algol.ttf") format("ttf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+body, h1, h2, h3, h4, h5 {
+  font-family: "Algol";
```

Comment:
Deze statement wordt hieronder overschreven (in regel 46). Hierdoor wordt het lettertype niet aangepast.

Het is hier beter om dezelfde stijl als in `dies.scss` aan te houden

```scss
$font-family-base: "Algol";
$headings-font-family: "Algol";
```